Apps/XDFBrowser/ui_mainwindow.h

```diff
@@ -1,130 +0,0 @@
-/********************************************************************************
```

Comment:
Will be automatically generated from the .ui-file (even from within Visual Studio)